### PR TITLE
multiple fixes for build/host

### PR DESCRIPTION
**Pull request by bgruening**
* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
@corburn, @dpryan79 can I get a review. I canceled the build I hope this PR improves recipes even if they do not compile and would like to merge it and build on bulk.

**Comment by dpryan79** [comment]
@bgruening Yes! That should fix at least a quarter of the failures.

File: recipes/blat/meta.yaml [hunk 1]
```diff
@@ -14,8 +14,6 @@ build:
 
 requirements:
   build:
-    - libpng
-    - zlib
   run:
```
Comment (dpryan79) [inline recipes/blat/meta.yaml]: Is this recipe even needed? It's probably identical to `ucsc-blat`.

Comment (bgruening) [reply recipes/blat/meta.yaml]: Probably not, I will remove it.

**Pull request merged**